client/scss/components/_listing.scss

```diff
@@ -697,23 +697,5 @@ td.ord {
 table.listing {
   th.ordered {
     color: $color-teal;
-
-    &.ascending {
-      &:before {
-        content: '\25B2'; // up arrow
-        display: inline-block;
-        height: 100%;
-        vertical-align: middle;
-      }
-    }
-
-    &.descending {
-      &:before {
-        content: '\25BC'; // down arrow
-        display: inline-block;
-        height: 100%;
-        vertical-align: middle;
-      }
-    }
```

Comment:
Those have been replaced with the same `-after` ordering icons as other listings.